sample-apps/react-native/dogfood/src/components/ParticipantLayoutButton.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { Pressable, Text, Modal, StyleSheet, View } from 'react-native';
+import GridIconSvg from '../assets/GridIconSvg';
+import { appTheme } from '../theme';
+import { CallParticipantsViewProps } from '@stream-io/video-react-native-sdk';
```

Comment:
no need to export props from SDK for this

The integrators can do `React.ComponentProps<CallParticipantsView>["mode"]`

this is the reason why prop types are not exported in react-native for example..